RELEASE/scripts/autoscend/iotms/mr2017.ash

```diff
@@ -819,7 +819,7 @@ boolean kgb_getMartini(string page, boolean dontCare)
 	{
 		if(!dontCare)
 		{
-			auto_log_info("We did not initialize the briefcase this ascension, we can not care", "red");
+			auto_log_info("We did not initialize the briefcase this ascension, we cannot care", "red");
```

Comment:
I don't have a KGB so I don't have any horse in this game, but is it that we can't care or do we have the luxury of not caring? If it's the latter, then technically it was right because we set dontCare to true